src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutePayloadResult.cs

```diff
@@ -15,14 +15,20 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 // 
 
+using System;
 using Nethermind.Core.Crypto;
+using Newtonsoft.Json;
 
 namespace Nethermind.Merge.Plugin.Data
 {
     public class ExecutePayloadResult
     {
+        [JsonIgnore]
         public Keccak BlockHash { get; set; }
 
-        public VerificationStatus Status { get; set; } 
+        public string? Status => Enum.GetName(typeof(VerificationStatus), EnumStatus)?.ToUpper();
```

Comment:
can we make it inmutable and hardcode status instead of allocating strings each time we execute a getter?